### ISSUE TITLE
"Edit on GitHub" link on Read the Docs is broken

### ISSUE THREAD

**Issue by TravisCarden** [acquia/blt#2969]
The "Edit on GitHub" link at the top right of [the Read the Docs site](http://blt.readthedocs.io/en/latest/) points incorrectly to https://github.com/acquia/blt/edit/master/docs/index.md. I'd fix it myself, but I don't know anything about Read the Docs or what controls that link.

**Comment by ba66e77** [comment]
Yeah, I can't seem to locate where that's coming from either. I've pinged the [RTFD issue queue](https://github.com/rtfd/readthedocs.org/issues/4441) to get a pointer.

**Comment by danepowell** [comment]
I think this is due to the fact that mkdocs uses a clone of the readthedocs theme, not the actual readthedocs theme. Since it doesn't have proper readthedocs integration to automatically determine the active Github branch, it just assumes "master". I opened an upstream issue for this: https://github.com/mkdocs/mkdocs/issues/1578

Maybe there's some way to get away from mkdocs and have pure readthedocs functionality, but in the meantime the easiest fix is manually set the correct path. See the referenced PR.
